src/main/kotlin/view/ResultView.kt

```diff
@@ -0,0 +1,25 @@
+package view
+
+import domain.MinesWeeper
+
+object ResultView {
+
+    private const val BOARD_PRINT_STRING = "\n지뢰찾기 게임 시작"
+
+    private const val MINE_STRING = "*"
+
+    private const val BASIC_STRING = "C"
+
+    fun printGameBoard(width: Int, minesWeeper: MinesWeeper) {
+        println(BOARD_PRINT_STRING)
+        val chucks = minesWeeper.boards.sortedBy { it.location.x }.sortedBy { it.location.y }.chunked(width)
+        for (chunk in chucks) {
+            println(chunk.joinToString(" ") { getPrintString(it.isMine) })
+        }
+    }
+
+    private fun getPrintString(isMine: Boolean): String {
+        if (isMine) return MINE_STRING
+        return BASIC_STRING
```

Comment:
아래와 같이도 구현해볼수 있을것 같아요 ~! 
```suggestion
        return if (isMine) MINE_STRING
        else BASIC_STRING
```

Reply:
else 예약어를 쓰지 않는다. 요구사항을 안지켜도되나요?!